website/docs/reference/resource-configs/snapshot_meta_column_names.md

```diff
@@ -67,17 +67,17 @@ In order to align with an organization's naming conventions, the `snapshot_meta_
 
 By default, dbt snapshots use the following column names to track change history using [Type 2 slowly changing dimension](https://en.wikipedia.org/wiki/Slowly_changing_dimension#Type_2:_add_new_row) records:
 
-| Field          | Meaning | Notes |
-| -------------- | ------- | ----- |
-| `dbt_valid_from` | The timestamp when this snapshot row was first inserted and became valid. | The value is affected by the [`strategy`](/reference/resource-configs/strategy). |
-| `dbt_valid_to`   | The timestamp when this row is no longer valid. |  |
-| `dbt_scd_id`     | A unique key generated for each snapshot row. | This is used internally by dbt. |
-| `dbt_updated_at` | The `updated_at` timestamp of the source record when this snapshot row was inserted. | This is used internally by dbt. |
-| `dbt_is_deleted` | A boolean value indicating if the record has been deleted. `True` if deleted, `False` otherwise. | Added when `hard_deletes='new_record'` is configured. |
+| Field          | <div style={{width:'250px'}}>Meaning</div> | Notes | [Example](#example) |
```

Comment:
Suggest leaving this out as a link since we link to it below, the formatting makes it difficult to read, and headers as links with no context as to where the link is taking the reader might hinder the experience. 
```suggestion
| Field          | <div style={{width:'250px'}}>Meaning</div> | Notes | Example |
```

Reply:
good call!